apps/gateway/src/api.e2e.ts

```diff
@@ -9,22 +9,25 @@ import { flushLogs, waitForLogs } from "./test-utils/test-helpers";
 const testModels = models
 	.filter((model) => !["custom", "auto"].includes(model.model))
 	.flatMap((model) => {
-		// Create the basic model entry
-		const baseModel = {
-			model: model.model,
-			providers: model.providers,
-		};
-
-		// Create entries for provider-specific model names that differ from the generic name
-		const providerSpecificModels = model.providers
-			.filter((p) => p.modelName && p.modelName !== model.model)
-			.map((p) => ({
-				model: p.modelName,
-				providers: [p],
+		const testCases = [];
+
+		if (model.providers.length === 1) {
+			testCases.push({
+				model: model.model,
+				providers: model.providers,
+			});
+		}
+
+		// Create entries for provider-specific requests using provider/model format
+		for (const provider of model.providers) {
+			testCases.push({
+				model: `${provider.providerId}/${model.model}`,
+				providers: [provider],
```

Comment:
this is great, but let's also still test a given model by model name if multiple providers are specified. additionally we want to also still test for each specific provider as you have already added here.